docs/CONFIG-VARS.md

```diff
@@ -109,6 +110,79 @@ subnet_ids = {
 }
 ```
 
+## IAM
+
+By default, three custom IAM Policies, and two custom IAM Roles (with Instance Profiles) are being created. If your site security protocol does not allow automatic creation of IAM resources, you can provide pre-created Roles using the following options:
```

Comment:
Is it really 3 custom IAM policies? I only see that viya4-iac-aws creates two. I also only see two custom policies described  below.

Reply:
corrected.